tests/e2e_test.go

```diff
@@ -4,71 +4,18 @@
 package tests
```

Comment:
i like how simple this overall `e2e_test` file is, to make sure I understand correctly, is the idea for other tests to share the same network? 

How does it work between multiple files, each having their own `Describe` containers

Reply:
Yup exactly. `Describe` blocks will run in serial, one after the other, and use the same environment set up in `BeforeSuite`

Reply:
If you have multiple files, each with their own describe container, how does ginkgo know to run in order/ which order to run in? Or is it run randomly ordered but in serial?

Reply:
I'm not sure how ordering is determined in this case, but Ginkgo suites and specs should generally be independent of one another, so order shouldn't matter. It would be great if there was a way to shuffle `Describe` blocks, but it unfortunately seems like that shuffling is only done on `It` specs (as long as the suite isn't ordered).